src/overlays/actors/ovl_En_Thiefbird/z_en_thiefbird.c

```diff
@@ -93,93 +146,1020 @@ static DamageTable D_80C13644 = {
+        Item_DropCollectible(globalCtx, &this->actor.world.pos, ITEM00_RUPEE_ORANGE);
+
+        for (i = 0; i < ARRAY_COUNT(D_80C13664); i++) {
+            for (j = 0; j < this->unk_196[i]; j++) {
+                Item_DropCollectible(globalCtx, &this->actor.world.pos, D_80C13664[i]);
+            }
+        }
+
+        Actor_MarkForDeath(&this->actor);
+    }
+}
+
+void func_80C11F6C(EnThiefbird* this, GlobalContext* globalCtx) {
+    SkelAnime_ChangeAnimTransitionRepeat(&this->skelAnime, &D_06000278, -4.0f);
+    func_80C10984(this, 15);
+    if (this->actor.colChkInfo.damageEffect != 3) {
+        this->actor.speedXZ = 4.0f;
+    } else {
+        this->actor.speedXZ = 0.0f;
+    }
+
+    if (this->actor.colChkInfo.damageEffect == 5) {
+        func_800BCB70(&this->actor, 0, 255, 0, 40);
+        Audio_PlayActorSound2(&this->actor, NA_SE_EN_COMMON_FREEZE);
+    } else if (this->actor.colChkInfo.damageEffect == 1) {
+        func_800BCB70(&this->actor, 0, 255, 0, 40);
+        Audio_PlayActorSound2(&this->actor, NA_SE_EN_COMMON_FREEZE);
+    } else {
+        func_800BCB70(&this->actor, 0x4000, 255, 0, 40);
+        Audio_PlayActorSound2(&this->actor, NA_SE_EN_THIEFBIRD_DAMAGE);
+    }
+
+    this->collider.base.acFlags &= ~AC_ON;
+    if (this->unk_3E8 == 0) {
+        this->unk_190 = -0x1000;
+        this->unk_192 = BINANG_ROT180(this->actor.yawTowardsPlayer);
+    } else {
+        this->unk_190 = Actor_PitchToPoint(&this->actor, &D_80C13920);
+    }
+
+    this->unk_18E = 40;
+    if (Rand_ZeroOne() < 0.9f) {
+        Item_DropCollectible(globalCtx, &this->actor.focus.pos, ITEM00_RUPEE_GREEN);
+    }
+
+    if (Rand_ZeroOne() < 0.6f) {
+        Item_DropCollectible(globalCtx, &this->actor.focus.pos, ITEM00_RUPEE_GREEN);
+    }
+
+    if (Rand_ZeroOne() < 0.3f) {
+        Item_DropCollectible(globalCtx, &this->actor.focus.pos, ITEM00_RUPEE_GREEN);
+    }
+    this->actionFunc = func_80C1215C;
+}
+
+void func_80C1215C(EnThiefbird* this, GlobalContext* globalCtx) {
+    f32 sp2C;
+
+    if (this->unk_18E != 0) {
+        this->unk_18E--;
+    }
+
+    if (this->unk_18C == 10) {
+        if (this->unk_18E < 38) {
+            func_80C114C0(this, globalCtx);
+            this->actor.speedXZ = 4.0f;
+        } else {
+            return;
+        }
+    }
+
+    SkelAnime_FrameUpdateMatrix(&this->skelAnime);
+    if (this->actor.bgCheckFlags & 8) {
+        this->unk_192 = this->actor.wallYaw;
+    } else if (this->unk_3E8 == 0) {
+        this->unk_192 = BINANG_ROT180(this->actor.yawTowardsPlayer);
+    } else {
+        this->unk_192 = Actor_YawToPoint(&this->actor, &D_80C13920);
+        this->unk_190 = Actor_PitchToPoint(&this->actor, &D_80C13920);
+    }
+
+    Math_SmoothStepToS(&this->actor.shape.rot.y, this->unk_192, 6, 0x1000, 0x100);
+    Math_SmoothStepToS(&this->actor.shape.rot.x, this->unk_190, 5, 0x100, 0x10);
+    sp2C = (this->unk_18E * 153.6f) + 2048.0f;
```

Comment:
```suggestion
    sp2C = (this->unk_18E * 153.6f) + 0x800;
```
May need `(f32)0x800`